recipes/typedload/conda_build_config.yaml

```diff
@@ -0,0 +1 @@
+python_min: '3.9'
```

Comment:
You don't need this file. You only need to define this if the package min is higher than conda-forge's min, which is related to Pyhton's EOL.

Reply:
I know, but without it, the windows building failed. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1106791&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de&l=735 `python_min' is undefined `
https://github.com/conda-forge/staged-recipes/issues/28445

Reply:
It should not. @beckermr do you knwo what is going on?

PS: B/c it is noarch, it is fine to fail in this case. Also, my comment above is about my preference, not a must to merge this.

Reply:
@beckermr @ocefpaf Since you're informed of the situation, I'll simply remove it.